### DISCUSSION TITLE
Display of tensors, Fortran ordering, etc.

### DISCUSSION THREAD

**Discussion by tgkolda** [sandialabs/pyttb#344] [General]
I have two issues (#338 & #340) that perhaps are better at this point as discussions. I think there is also some subtle connection here to the underlying ordering. I really dislike the numpy way of displaying arrays because it doesn't align the columns. But perhaps a bigger concern for me is that the bracketing gives the impression of a C-style ordering rather than an F-style ordering. I don't know that it's clear enough in the documentation that pyttb uses a F order under the hood. For anyone doing linear indexing into the tensor, this will be really critical to understand. I could even imagine stating this explicitly every time a tensor is printed.

**Comment by ntjohnson1** [comment]
We've merge a PR(s) to main to make the Fortran ordering more consistent when we return numpy arrays, print out the F ordering with the tensors, and print based on frontal slices. Took a little time to get things cleaned up. I'll close this for now but we can re-open if there are other conceptual things to discuss. Otherwise can open issues for any edge cases we may have missed.
